app-models/articles-models.js

```diff
@@ -4,7 +4,37 @@ const { checkUserExists } = require("../functions/user-check");
 
 exports.fetchArticleById = (article_id) => {
   return db
-    .query(`SELECT * FROM articles WHERE article_id = $1;`, [article_id])
+    .query(
+      `SELECT
+      articles.article_id,
+      articles.title,
+      articles.topic,
+      articles.author,
+      articles.body,
+      articles.created_at,
+      articles.votes,
+      articles.article_img_url,
+      COUNT(comments.comment_id)::INT AS comment_count
+    FROM
+      articles
+    LEFT JOIN
+      comments ON articles.article_id = comments.article_id
+    WHERE
+      articles.article_id = $1
+    GROUP BY
+      articles.article_id,
```

Comment:
We don't need to select every single column for GROUP BY. We would generally want to group by columns the that contain duplicate values - for example articles.article_id